### ISSUE TITLE
Where's the git history for each release?

### ISSUE THREAD

**Issue by henricook** [flyway/flyway#3778]
Hi everyone,

I'm trying to find the per-commit history for the 10.0.0 release so that I can isolate a breaking change and get some background on why it was made so I can figure out how to adjust my code, which was using `PostgreSQLConfigurationExtension` to `setTransactionalLock(false)`.

On the `main` branch though I only see one big commit for the entire release with hundreds of changed files, but not individual commits. Are they available somewhere please?

I tried checking the contributors guidelines from the main page but they 404, and the tags have the same fat commits.

**Comment by Barry-RG** [comment]
Unfortunately due to the way Flyway is currently being developed, we have a squash commit for every release. This has been the way since about 9.0.2. 
We are working towards returning to more fine-grain commits in this repository.

**Comment by estomagordo** [comment]
I also find it weird that the official release notes say this:

> Modularizeddatabase support in Flyway to allow greater flexibility. This includes; DB2, Derby, HSQLDB, Informix, PostgreSQL, CockroachDB, Redshift, SAP HANA, Snowflake and Sybase ASE. See Database Support page for your database for module dependency.

But there are no links or explanations to what this means. I assume that `PostgreSQLConfigurationExtension` now resides in a different library, somewhere?

**Comment by henricook** [comment]
Correct, it's not clear at all from the release notes but I found a little readme in the super change set: https://github.com/flyway/flyway/commit/a7ce51e5939ad113fa1f14c6ada69b77f124ff60#diff-41027e9d261d59be11cbe5a8e18905dcc71f3529a4f4c78c7e10f5de800a931dR12

TL;DR - you need to add a new import to your build.sbt for e.g. the new flyway postgres module, and then the package name has changed to remove `something.internal` but it's otherwise the same.

Apologies for brevity, on mobile.

**Comment by estomagordo** [comment]
I don't quite understand what I need to do in order to continue (Kotlin, Gradle), but that is more to do with my level of competency than your brevity :)

**Comment by henricook** [comment]
Back on a machine now!

At the top of the file where I am using `PostgresqlConfigurationExtension` I had to change the import:

from: `import org.flywaydb.core.internal.database.postgresql.PostgreSQLConfigurationExtension`
to: `import org.flywaydb.database.postgresql.PostgreSQLConfigurationExtension`

And to make `import org.flywaydb.database.postgresql` available I had to add a new dependency in my `build.sbt` (pom file equivalent, not sure what the Kotlin/Gradle mapping is)

from: 
```
"org.flywaydb"                             % "flyway-core"               % "10.0.0",
```

to:

```
"org.flywaydb"                             % "flyway-core"                                         % "10.0.0",
"org.flywaydb"                             % "flyway-database-postgresql"               % "10.0.0",
```
